SokolTextGame/Commands.cs

```diff
@@ -36,10 +36,20 @@ public class Look : ICommand
     {
         public void Execute(string[] words, World world)
         {
-            if (string.Join(" ", words) == "look") Console.Write("Look... What's next? You can look at your weapon\n");
+            var locObj = world.CurrentLocation().ObjectOnLocation;
+            if (string.Join(" ", words) == "look") Console.Write("Look... What's next? You can look at your weapon or look around\n");
             else if (string.Join(" ", words) == "look at") Console.Write("Look at... what?\n");
             else if (string.Join(" ", words) == "look at my weapon") Console.Write(world?.player?.Weapon.Description);
-            else Console.Write("I know the command \"look at my weapon\"\n");
+            else if (string.Join(" ", words.Take(2)) == "look at " && locObj?.Name == string.Join(" ",words.Skip(2))) Console.Write(locObj.Description);
```

Comment:
Also please add a condition for when the player says `look at <object>` but the object does not exist. For now it just simply says the generic "I know the command..." in that case.